packages/beacon-node/src/sync/sync.ts

```diff
@@ -78,15 +78,18 @@ export class BeaconSync implements IBeaconSync {
     this.unknownBlockSync.close();
   }
 
-  getSyncStatus(): SyncingStatus {
+  async getSyncStatus(): Promise<SyncingStatus> {
     const currentSlot = this.chain.clock.currentSlot;
+    const elOffline = await this.chain.executionEngine.isOffline();
```

Comment:
We should deeply review the implications of adding this call here, sync is delicate. Also, what's somewhat concerning is that this is a different code path than other executionEngine consumers. It's possible that:
- block processing attempts to call executionEngine and errors because it's offline
- this getSyncStatus calls eth_chainId and succeeds, considering el online
How to resolve this inconsistency? It can happen both ways